doc/CMakeLists.txt

```diff
@@ -1,27 +1,7 @@
-if(HIGH_FIVE_DOCUMENTATION)
-find_package(Pandoc REQUIRED)
-
-configure_file(build.sh.cmake build.sh @ONLY)
-
-add_custom_target(doc ALL DEPENDS
-	COMMAND sh ${CMAKE_CURRENT_BINARY_DIR}/build.sh
-	DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/build.sh
-)
-
-install(
-	FILES ${CMAKE_CURRENT_BINARY_DIR}/index.html
-    DESTINATION ${HTML_INSTALL_DIR}
-)
-install(
-	DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/src
-    DESTINATION ${HTML_INSTALL_DIR}
-)
-
 find_package(Doxygen)
 if(Doxygen_FOUND)
     configure_file(${CMAKE_CURRENT_SOURCE_DIR}/Doxyfile ${CMAKE_CURRENT_BINARY_DIR}/Doxyfile @ONLY)
-    add_custom_target(doxygen ALL COMMAND Doxygen::doxygen ${CMAKE_CURRENT_BINARY_DIR}/Doxyfile)
```

Comment:
i should name the target **doc** instead of **doxygen**, no?

**make doc** vs **make doxygen**

Reply:
I would go for `html`.
See https://www.gnu.org/prep/standards/html_node/Standard-Targets.html

Reply:
I never see this convention respected anywhere.

Reply:
> I never see this convention respected anywhere.

I don't understand your point?

Reply:
In the last meeting about coding conventions in BBP (I was in vacations), it was suggested to have a make target called `doc`:
See https://bbpteam.epfl.ch/project/spaces/display/QA/Workshop+Coding+Guidelines

Reply:
I mean that **make html** sounds weird for me, and I can't remember a project using that to build the doc.

Reply:
I mean that **make html** sounds weird for me, and I can't remember a project using that to build the doc.

Reply:
The fact that there are projects that don't follow the convention doesn't make it less valid.
I can name a few that follow it: sphinx Makefiles and all GNU projects.

Like I said above, the target may be named `doc` to meet BBP convention.

Reply:
Ok I put doc.